### ISSUE TITLE
Windows build failing ?

### ISSUE THREAD

**Issue by drupol** [shivammathur/setup-php#172]
Hi,

I noticed since today that we are unable to have the `xdebug` extension and PHP 7.1.33 on Windows machine.
It was working fine 2 days ago.

See a working build: https://github.com/ecphp/cas-lib/runs/417225443?check_suite_focus=true#step:3:1

See a failing build: https://github.com/ecphp/cas-lib/runs/419407382?check_suite_focus=true#step:3:1

**Comment by shivammathur** [comment]
@mlocati Please have a look at this.
For `nts` PHP versions `Xdebug` installation is failing with this error
```
PS > Install-PhpExtension -Extension xdebug -MinimumStability stable -Path $php_dir
Install-PhpExtension : The variable '$finalDllName' cannot be retrieved because it has not been set.
At line:1 char:1
+ Install-PhpExtension -Extension xdebug -MinimumStability stable -Path ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (finalDllName:String) [Install-PhpExtension], RuntimeException
+ FullyQualifiedErrorId : VariableIsUndefined,Install-PhpExtension
```

**Comment by mlocati** [comment]
@shivammathur https://github.com/mlocati/powershell-phpmanager/pull/53 should fix that issue

**Comment by shivammathur** [comment]
@mlocati Thanks for the quick fix.

**Comment by shivammathur** [comment]
@mlocati Please publish the release with  mlocati/powershell-phpmanager#53

**Comment by mlocati** [comment]
Done, I just released the [new version 1.22.4.301](https://www.powershellgallery.com/packages/PhpManager/1.22.4.301)

**Comment by shivammathur** [comment]
@mlocati Thanks

@drupol Fixed in mlocati/powershell-phpmanager#53. Here is a [test workflow](https://github.com/shivammathur/test-setup-php/actions/runs/33505029)

**Comment by drupol** [comment]
Impressive. Thanks guys !!!
